soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4620,6 +4620,85 @@ void CreateScrubMessages() {
     }
 }
 
+void CreateRupeeMessages() {
+    CustomMessageManager* customMessageManager = CustomMessageManager::Instance;
+    customMessageManager->AddCustomMessageTable(Randomizer::rupeeMessageTableID);
+    const std::vector<u8> rupees = { TEXT_BLUE_RUPEE, TEXT_RED_RUPEE, TEXT_PURPLE_RUPEE, TEXT_HUGE_RUPEE };
+    std::string rupeeText = " ";
+    for (u8 rupee : rupees) {
+        switch (rupee) {
+            case TEXT_BLUE_RUPEE:
+                rupeeText = "\x05\x03 5 %RUPEE%\x05\x00";
```

Comment:
That's fair. That said, we should probably decide on a symbol that can be used for these kinds of "variables" in the future as well and reserve it for these kinds of situations, so I'd like your input on what sign would fit the best here. 

I don't think it super matters as long as we're consistent. Something like #RUPEE# works for me.

Reply:
right now `@` is player name, maybe we can just use `$` for rupees for now

Reply:
but yeah, https://github.com/HarbourMasters/Shipwright/issues/1002

Reply:
Even with #1002 in mind, it might make more sense to have generic "variables" that use a specific symbol + name. Otherwise we're going to run out of special characters sooner than later, and feels much more readable than a random special character. Then very common ones can have their own symbol like player name and color.

Anyway, I'll swap it to #RUPEE# for this PR, and we can revisit this with #1002 with whatever we decide on then.